app/views/partials/lib/commands/posts/cache_score.liquid

```diff
@@ -0,0 +1,11 @@
+{% liquid
+  function post = 'lib/queries/posts/find', id: post_id
+
+  assign object = '{}' | parse_json
+  hash_assign object['id'] = post.id
+  hash_assign object['c__score'] = post.upvote_profile_ids.size | minus: post.downvote_profile_ids.size | default: 0
```

Comment:
we should use `decrement` atomic operation for this one, and `increment` for upvote` instead of calculating the value in the liquid, will create a separate card

Reply:
@Slashek I my opinion this is good. This is only cache, we can't mutate cache, we have to calculate it from source data.

Reply:
if two people will upvote within a very short interval, this value will be wrong. Not critical without many users, but hopefully it will eventually be a problem.